images/macos/macos-11-Readme.md

```diff
@@ -1,52 +1,46 @@
 | Announcements |
 |-|
-| [[all OSs] Default Nodejs version will be set to 16 on November, 29](https://github.com/actions/virtual-environments/issues/4446) |
 | [macOS-latest workflows will use macOS-11](https://github.com/actions/virtual-environments/issues/4060) |
 ***
 # macOS 11.6 info
 - System Version: macOS 11.6.1 (20G224)
 - Kernel Version: Darwin 20.6.0
-- Image Version: 20211127.1
+- Image Version: 20211207.1
 
 ## Installed Software
 ### Language and Runtime
 - .NET SDK 2.1.302 2.1.403 2.1.507 2.1.607 2.1.701 2.1.818 3.1.101 3.1.201 3.1.302 3.1.415 5.0.102 5.0.202 5.0.302 5.0.403
 - Bash 3.2.57(1)-release
-- Clang/LLVM 13.0.0 is available on `'$(brew --prefix llvm)/bin/clang'`
-- Clang/LLVM 13.0.0 is default
-- gcc-9 (Homebrew GCC 9.4.0) 9.4.0 - available by `gcc-9` alias
-- gcc-10 (Homebrew GCC 10.3.0) 10.3.0 - available by `gcc-10` alias
-- gcc-11 (Homebrew GCC 11.2.0_3) 11.2.0 - available by `gcc-11` alias
-- GNU Fortran (Homebrew GCC 9.4.0) 9.4.0 - available by `gfortran-9` alias
-- GNU Fortran (Homebrew GCC 10.3.0) 10.3.0 - available by `gfortran-10` alias
-- GNU Fortran (Homebrew GCC 11.2.0_3) 11.2.0 - available by `gfortran-11` alias
+- Clang/LLVM 13.0.0 is default Clang/LLVM 13.0.0 is available on `'$(brew --prefix llvm)/bin/clang'`
+- gcc-9 (Homebrew GCC 9.4.0) 9.4.0 - available by `gcc-9` alias gcc-10 (Homebrew GCC 10.3.0) 10.3.0 - available by `gcc-10` alias gcc-11 (Homebrew GCC 11.2.0_3) 11.2.0 - available by `gcc-11` alias
+- GNU Fortran (Homebrew GCC 9.4.0) 9.4.0 - available by `gfortran-9` alias GNU Fortran (Homebrew GCC 10.3.0) 10.3.0 - available by `gfortran-10` alias GNU Fortran (Homebrew GCC 11.2.0_3) 11.2.0 - available by `gfortran-11` alias
```

Comment:
@V-Zabayrachny is this caused by your changes here? https://github.com/actions/virtual-environments/pull/4587

Reply:
Looks like not as the issue appeared on the 12\03 first time.
Created internal issue for investigation https://github.com/actions/virtual-environments-internal/issues/3081